backend/app/controllers/admin/options_controller.rb

```diff
@@ -1,4 +1,17 @@
 module Admin
   class OptionsController < Admin::ApplicationController
+    def create
+      resource = resource_class.new(resource_params)
+      authorize_resource(resource)
+
+      if resource.save
+        flash[:notice] = t('administrate.option.created.success')
+        redirect_to admin_options_path
```

Comment:
I believe you don't need to override the whole method, I think you can override only https://github.com/thoughtbot/administrate/blob/fd83fbbb125446f782d9713229c69dcc1d826843/app/controllers/administrate/application_controller.rb#L88

Reply:
@edimossilva I tried to override the `after_resource_created_path` method. But it wasn`t working. Finally I realized where the **problem** is:
- In this version of this gem that we are using `after_resource_created_path` method is not implemented yet. 

**Solutions:**
- We can update the gem **OR**
- override the `create` method that hasn't  `after_resource_created_path` method.

Reply:
Hi @PolinaStamenova , lets do the opposite, let me know what do you prefer and why?

Reply:
I agree 100%, nice explanation!

Reply:
The bundler version in our project is >= 1.14, so to update the gem we need to add the flag `--conservative`: 
```
bundle update --conservative --source administrate
```